nemo/collections/llm/recipes/deepseek_v3.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Callable, Optional
+
+import lightning.pytorch as pl
+import nemo_run as run
+
+from nemo.collections.llm.api import finetune, pretrain
+from nemo.collections.llm.gpt.data.packed_sequence import PackedSequenceSpecs
+from nemo.collections.llm.gpt.model.deepseek import DeepSeekModel, DeepSeekV3Config
+from nemo.collections.llm.peft import PEFT_STR2CLS
+from nemo.collections.llm.recipes.finetune_default import default_finetune_recipe
+
+NAME = "deepseek_v3"
+
+
+@run.cli.factory(name=NAME)
+def model() -> run.Config[pl.LightningModule]:
+    """
+    Factory function to create a DeepSeek-V3 (671B) model configuration.
+
+    Returns:
+        run.Config[pl.LightningModule]: Configuration for the DeepSeek V3 model.
+
+    Examples:
+        CLI usage:
+            $ nemo llm pretrain model=deepseek_v3 ...
+
+        Python API usage:
+            >>> model_config = model()
+            >>> print(model_config)
+    """
+    conf = run.Config(DeepSeekV3Config)
+    return run.Config(DeepSeekModel, config=conf)
+
+
+@run.cli.factory(target=pretrain, name=NAME)
```

Comment:
Let's comment these out for now, since otherwise it will show up in the CLI

Reply:
good catch!